FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.tsx

```diff
@@ -1,7 +1,101 @@
 import React, { FunctionComponent } from 'react';
 import { Col, Container, Row } from 'react-bootstrap';
 import styles from './MemberInfoCard.module.scss';
+import Card from 'react-bootstrap/Card';
+import { contactIcons } from '@icons';
+import { Link, FilledImage } from '@basics';
+import { Body1, Body2, Caption } from '@components/basics';
+import cn from 'classnames';
 
-const MemberInfoCard: FunctionComponent = () => <Container></Container>;
+export interface MemberInfoCardProps {
+  name: string;
+  imgSrc: string;
+  title: string;
+  quote: string | { number: number; length: string };
+  website?: string;
+  email?: string;
+  linkedIn?: string;
+}
+
+const MemberInfoCard: FunctionComponent<MemberInfoCardProps> = ({
+  name,
+  imgSrc,
+  title,
+  quote,
+  website,
+  email,
+  linkedIn,
+}) => {
+  const firstName: string = name.split(' ')[0];
+  let team: string = title.toLowerCase();
+  team === 'designer' && (team = 'design');
```

Comment:
This line is confusing. Generally, we don't want "side effects" to happen when we do '&&' or '||'. Perhaps a better way to code this would be to use an if statement. Alternatively, you can do something like the following:
const lowerCasedTitle = title.toLowerCase()
const team = lowerCasedTitle  === 'designer' ? 'design' : lowerCasedTitle;

Also, note that it's common practice to use && and || to render (as you did inside contactCol variable below) but elsewhere, we tend not to use it.